kornia/geometry/warp/homography_warper.py

```diff
@@ -45,6 +46,30 @@ def warp_grid(grid: torch.Tensor, src_homo_dst: torch.Tensor) -> torch.Tensor:
     return flow.view(batch_size, height, width, 2)  # NxHxWx2
 
 
+def warp_grid3d(grid: torch.Tensor, src_homo_dst: torch.Tensor) -> torch.Tensor:
```

Comment:
I think this is the same as [affine_grid](https://pytorch.org/docs/stable/nn.functional.html#torch.nn.functional.affine_grid)
 for input transform 3x4

Reply:
for the homography we had to create this over-plaiting code since affine_grid only accepts 2x3 or 3x4 but for the case of 4x4 I think it can be used since last row it's [0, 0, 0, 1]. /cc @ducha-aiki

Reply:
It is the same, yes.